src/redist/targets/GenerateBundledVersions.targets

```diff
@@ -359,6 +359,7 @@ Copyright (c) .NET Foundation. All rights reserved.
     <NetCoreRoot Condition="'%24(NetCoreRoot)' == ''">%24([MSBuild]::NormalizePath('%24(MSBuildThisFileDirectory)..\..\'))</NetCoreRoot>
     <NetCoreTargetingPackRoot Condition="'%24(NetCoreTargetingPackRoot)' == ''">%24([MSBuild]::EnsureTrailingSlash('%24(NetCoreRoot)'))packs</NetCoreTargetingPackRoot>
 
+    <MicrosoftNetCompilersToolsetPackageVersion>$(MicrosoftNetCompilersToolsetPackageVersion)</MicrosoftNetCompilersToolsetPackageVersion>
```

Comment:
Isn't this inside raw `CDATA`? Won't you end up with a literal string like `<MicrosoftNetCompilersToolsetPackageVersion>$(MicrosoftNetCompilersToolsetPackageVersion)</MicrosoftNetCompilersToolsetPackageVersion>`? I assume you wanted the file to contain an actual version like `<MicrosoftNetCompilersToolsetPackageVersion>1.2.3</MicrosoftNetCompilersToolsetPackageVersion>`